lib/chef/knife/clc_server_create.rb

```diff
@@ -281,9 +290,9 @@ def check_server_platform
           windows_platform = server['os'] =~ /windows/
         end
 
-        if windows_platform
-          errors << 'Bootstrapping is available for Linux platform only'
-        end
+        # if windows_platform
```

Comment:
This method should be used to choose bootstrapping platform. Also we should probably provide an option to override platform manually in case if it backfires.